### DISCUSSION TITLE
Any projects using Diesel?

### DISCUSSION THREAD

**Discussion by musjj** [diesel-rs/diesel#4273] [Q&A]
It would be nice if there's an overview of real-world projects using Diesel. https://crates.io is a solid example, but I'd like to know if there's more out there.

Something like SeaORM's [COMMUNITY.md](https://github.com/SeaQL/sea-orm/blob/master/COMMUNITY.md) would be awesome.

**Comment by weiznich** [comment]
You can see a list of all GitHub projects depending on diesel here: https://github.com/diesel-rs/diesel/network/dependents

The problem with having such a Community.md file is that it either quickly be outdated or it is a significant amount of work to update the document from time to time.
